src/ImageSharp/Formats/Webp/Lossless/HistogramEncoder.cs

```diff
@@ -439,18 +450,17 @@ private static bool HistogramCombineStochastic(List<Vp8LHistogram> histograms, i
                 // Make sure the index order is respected.
                 if (p.Idx1 > p.Idx2)
                 {
-                    int tmp = p.Idx2;
-                    p.Idx2 = p.Idx1;
-                    p.Idx1 = tmp;
+                    (p.Idx1, p.Idx2) = (p.Idx2, p.Idx1);
                 }
 
                 if (doEval)
                 {
                     // Re-evaluate the cost of an updated pair.
-                    HistoListUpdatePair(histograms[p.Idx1], histograms[p.Idx2], stats, bitsEntropy, 0.0d, p);
-                    if (p.CostDiff >= 0.0d)
+                    HistoListUpdatePair(histograms[p.Idx1], histograms[p.Idx2], stats, bitsEntropy, 0D, p);
+
+                    if (p.CostDiff >= 0D)
```

Comment:
Re: `0D` before the literals were lowercase which is IMO more readable (`0d` <-> `0D`).
Anymway it should be consistent throughout the code-base.

Reply:
I have an irrational dislike for lowercase suffix type identifiers for `F`, `D`, `L`, and `M`. I cannot explain why as I use `u` at times.